fluss-flink/fluss-flink-common/src/main/java/org/apache/fluss/flink/lake/LakeSplitSerializer.java

```diff
@@ -51,6 +51,7 @@ public LakeSplitSerializer(SimpleVersionedSerializer<LakeSplit> sourceSplitSeria
     }
 
     public void serialize(DataOutputSerializer out, SourceSplitBase split) throws IOException {
+        out.writeInt(getVersion());
```

Comment:
we don't need to write `LakeSplitSerializer`. It's a part of `SourceSplitSerializer`, whose version is already tracked by flink framework.